.github/workflows/release.yml

```diff
@@ -44,14 +44,8 @@ jobs:
 
       - name: Publish
         if: "!github.event.release.prerelease"
-        working-directory: ./dist
-        run: |
-          npm pkg delete scripts.prepare
```

Comment:
Did we remove this line, since npm doesn't already include it in release?

Reply:
It includes it but does not run the script when doing install. But now I learned that it runs the script for git installs etc. I am adding this part back